walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -40,19 +41,71 @@
     ArrayList<Event> sortedEvents = new ArrayList<>(events);
     Collections.sort(sortedEvents, Event.ORDER_BY_START);
 
+    Collection<Event> optionalEvents = new ArrayList<>();
+
+    ListIterator<Event> eventsIterator = sortedEvents.listIterator();
+    while (eventsIterator.hasNext()) {
+      Event event = eventsIterator.next();
```

Comment:
I wanted to modify sortedEvents in place instead of making another ArrayList. If you (or anyone else) feel like this reduces readability though, I could definitely change it.

Reply:
I see I see. Yah its fine with me!